lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,64 @@
+class TicTacToe
+  attr_accessor :board
+  def initialize(board)
+    @board = board
+  end
+
+  def draw_board
+    puts "\n"
+    puts "#{@board[0]}  |#{@board[1]} |#{@board[2]}"
+    puts '---------'
+    puts "#{@board[3]}  |#{@board[4]} |#{@board[5]}"
+    puts '---------'
+    puts "#{@board[6]}  |#{@board[7]} |#{@board[8]}"
+    puts "\n"
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
- [ ] remove these comments. You should only have `return` statements in thifs file.